containers/ome-tiff-tiler/Dockerfile

```diff
@@ -0,0 +1,38 @@
+ARG BUILD_IMAGE=gradle:6.2.1-jdk8
+
+#
```

Comment:
Did we get all of this from somewhere? Could we cite a source, either a person or URL?